src/components/draggable/draggable.stories.tsx

```diff
@@ -46,6 +46,24 @@ export const DefaultStory: Story = () => (
 );
 DefaultStory.storyName = "Default";
 
+export const FlexDirectionStory: Story = () => (
```

Comment:
suggestion: it might be worth changing the children a bit here so when a user drags it's obvious the order has been updated etc